src/go/rdctl/cmd/setup.go

```diff
@@ -0,0 +1,27 @@
+package cmd
+
+import (
+	"github.com/rancher-sandbox/rancher-desktop/src/go/rdctl/pkg/autostart"
+	"github.com/spf13/cobra"
+)
+
+var setupSettings struct {
+	AutoStart bool
+}
+
+var setupCmd = &cobra.Command{
+	Hidden: true,
+	Use:    "setup",
+	Short:  "Configure the system without modifying settings",
+	RunE: func(cmd *cobra.Command, args []string) error {
+		if cmd.Flags().Changed("auto-start") {
+			return autostart.EnsureAutostart(setupSettings.AutoStart)
+		}
```

Comment:
What's it mean if no flag is specified? I think this should give an error message, rather than be a no-op. The user typed a command, and should expect either to trigger some kind of change or get an error message